code/api/src/modules/order/types.js

```diff
@@ -14,12 +15,16 @@ const OrderType = new GraphQLObjectType({
     id: {type: GraphQLInt },
     user: { type: UserType },
     crate: { type: CrateType },
-    productDeliveries: { type: new GraphQLList( ProductDeliveryType )},
+<<<<<<< HEAD
+    productDeliveries: { type: new GraphQLList(ProductDeliveryType) },
+=======
+>>>>>>> Update types and order query
+    products: { type: new GraphQLList(ProductType) },
```

Comment:
It looks like there might be some leftover rebase or merge notes in here? @jdcarey128